debian/libspandsp3.install

```diff
@@ -0,0 +1 @@
+debian/tmp/usr/lib/*/spandsp3/libspandsp3.so.*
```

Comment:
we should be doing libspandsp.so.X where x is bigger than what the other packages are doing

Reply:
If we do not change library name, pkg-config file would be named as the one that stock spandsp package bring. (spandsp.pc)

If we do manually rename spandsp.pc to spandsp3.pc, there would be still issue with pkg-config looking-up for proper library. (`Libs: -L${libdir} -lspandsp` vs `Libs: -L${libdir} -lspandsp3`)

Only way that I can see to solve this, is to set major version to some X, make library package coinstallable with upstream package, make devel package replace upstream package so pkg-config would work.